libraries/chain/db_market.cpp

```diff
@@ -641,7 +640,7 @@ int database::match( const limit_order_object& bid, const call_order_object& ask
    bool before_core_hardfork_184 = ( maint_time <= HARDFORK_CORE_184_TIME ); // something-for-nothing
    // TODO remove when we're sure it's always false
```

Comment:
What about this TODO?

Reply:
I was able to clean up the warnings for 342 and 184, as they never appeared in the log for both testnet nor mainnet.